.travis.yml

```diff
@@ -5,4 +5,4 @@ before_install:
   - npm install -g gulp
   - npm install
 script:
-  - gulp
+  - gulp release
```

Comment:
So this will add unit test now to travis right ?

Reply:
@TimothyAsir Yes. gulp release task includes unit testing as well.